packages/components/src/components/install-guide/record-instructions/Ruby.vue

```diff
@@ -1,69 +1,91 @@
-        >.
-      </p>
+      <section class="recording-method">
+        <h3>
+          <i class="header-icon"><TestsIcon /></i>Tests recording<span class="recommended-badge"
+            >recommended</span
+          >
+        </h3>
+        <p>
+          When you run your {{ testFramework.name }} tests with AppMap enabled, an AppMap will be
+          created for each test.
+        </p>
+        <p>
+          <template v-if="testFramework.name.toLowerCase() == 'minitest'">
+            Run Minitest tests:
+            <v-code-snippet clipboard-text="DISABLE_SPRING=true rails test" />
+          </template>
+          <template v-if="testFramework.name.toLowerCase() == 'rspec'">
+            Run Rspec tests:
+            <v-code-snippet clipboard-text="DISABLE_SPRING=true rspec" />
+          </template>
+        </p>
+        <p>
+          For more information, visit
+          <a
+            href="https://appmap.io/docs/reference/appmap-ruby.html#tests-recording"
+            target="_blank"
+            >AppMap docs - Ruby Tests recording</a
+          >.
+        </p>
+      </section>
     </template>
     <template v-else>
       <VTestsPrompt frameworks="RSpec or Minitest" />
     </template>
+    <br />
     <template v-if="webFramework">
-      <h3>Requests recording</h3>
-      <p>
-        When your application uses {{ this.webFramework.name }}, and you run your application with
-        AppMap enabled, HTTP server requests recording is enabled. To record requests, first run
-        your application:
-      </p>
-      <p>
-        Start your Rails server:
-        <v-code-snippet clipboard-text="DISABLE_SPRING=true rails server" />
-      </p>
-      <p>
-        Interact with your application, through its user interface and/or by making API requests
-        using a tool such as Postman. An AppMap will be created for each HTTP server request that's
-        served by your app.
-      </p>
-      <p>
-        For more information, visit
-        <a
-          href="https://appmap.io/docs/reference/appmap-ruby.html#requests-recording"
-          target="_blank"
-          >AppMap docs - Ruby Requests recording</a
-        >.
-      </p>
+      <section class="recording-method">
+        <h3>
+          <i class="header-icon"><RequestsIcon /></i>Requests recording
+        </h3>
+        <p>
+          When your application uses {{ this.webFramework.name }}, and you run your application with
+          AppMap enabled, HTTP server requests recording is enabled. To record requests, first run
+          your application:
+        </p>
+        <p>
+          Start your Rails server:
+          <v-code-snippet clipboard-text="DISABLE_SPRING=true rails server" />
+        </p>
+        <p>
+          Interact with your application, through its user interface and/or by making API requests
+          using a tool such as Postman. An AppMap will be created for each HTTP server request
+          that's served by your app.
```

Comment:
See comments on other similar area elsewhere in the PR

Reply:
Not sure what you are pointing out here. Could you be more specific?